packages/tradershub/src/features/cfd/screens/CFDCompareAccounts/CompareAccountsCard.tsx

```diff
@@ -38,27 +29,11 @@ const CompareAccountsCard = ({
                         </Text>
                     </div>
                 )}
-                <CompareAccountsTitleIcon
-                    isDemo={isDemo}
-                    marketType={marketType}
-                    platform={platform}
-                    shortCode={shortCode}
-                />
-                <CompareAccountsDescription
-                    isDemo={isDemo}
-                    isEuRegion={isEuRegion}
-                    marketType={marketType}
-                    shortCode={shortCode}
-                />
-                <InstrumentsLabelHighlighted
-                    isDemo={isDemo}
-                    isEuRegion={isEuRegion}
-                    marketType={marketType}
-                    platform={platform}
-                    shortCode={shortCode}
-                />
-                {isEuUser && (
-                    <div className='relative text-center top-[5px]'>
+                <CompareAccountsTitleIcon marketType={marketType} platform={platform} shortCode={shortCode} />
+                <CompareAccountsDescription marketType={marketType} shortCode={shortCode} />
+                <InstrumentsLabelHighlighted marketType={marketType} platform={platform} shortCode={shortCode} />
```

Comment:
Remove prop drilling

Reply:
Nice Initiative
This Prop drilling was because we were using observer for store and we dont want to wrap every component to observer.
Now it make sense to remove these props